pyanaconda/modules/payloads/payload/flatpak/flatpak_manager.py

```diff
@@ -50,6 +50,12 @@
 __all__ = ["FlatpakManager"]
 
 
+# FIXME: Workaround for https://issues.redhat.com/browse/RHEL-85624 - remove when fixed
```

Comment:
Any idea if this was perhaps resolved in upstream in the meantime ?

Reply:
It wasn't. They know about it but it is not a priority for now as the workaround works right now and the focus is get this out.

Reply:
OK, thanks for checking - is there perhaps an upstream bug we could link from there for future reference ?